FHEM/00_SIGNALduino.pm

```diff
@@ -2926,11 +2918,6 @@ sub SIGNALduino_Parse_MN {
   my %msg_parts = SIGNALduino_Split_Message($rmsg, $hash->{NAME});
   
   my $rawData = (substr %msg_parts{rawData},0,1 eq q[Y]) ? substr(%msg_parts{rawData},1) : $msg_parts{rawData};
```

Comment:
Hier mäkelt Perl:
`PERL WARNING: %msg_parts{"rawData"} in scalar context better written as $msg_parts{"rawData"}`

Reply:
Ja das stimmt auch...

Reply:
Der Code ist angepasst